pkg/loop/internal/chain_reader.go

```diff
@@ -0,0 +1,135 @@
+package internal
+
+import (
+	"context"
+	jsonv1 "encoding/json"
+	"fmt"
+
+	jsonv2 "github.com/go-json-experiment/json"
+
+	"github.com/fxamacker/cbor/v2"
+
+	"github.com/smartcontractkit/chainlink-relay/pkg/loop/internal/pb"
```

Comment:
You should be able to do a plain find/replace on chainlink-relay -> chainlink-common